### PR TITLE
Monday morning, from number is invalid

### PR DESCRIPTION
**Pull request by AlexPerson**

### CODE REVIEW

File: lib/takeaway_class.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+	attr_reader :menu, :order, :actualcost
+
+	def initialize
+		@menu = {"Cheeseburger" => 1,
+			"Pizza" => 2,
+			"Fries" => 3
+		}
+		@actualcost = 0
+	end
```
Comment (ojlamb) [inline lib/takeaway_class.rb]: You might want to separate responsibilities of your menu and order into different classes.

Comment (ojlamb) [inline lib/takeaway_class.rb]: Ruby convention is 2 space indentation. Clean and readable code is really important moving forward. Try adjusting the settings in your text editor .

File: lib/test.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+account_sid = 'ACf8addc5058d1be8f246fd75fa4ce9013'
+auth_token = 'bb5aa7da4e982b37aac058a31d962549'
```
Comment (ojlamb) [inline lib/test.rb]: use .gitignore or an environment variable to hide tokens before pushing to git.

**Comment by tansaku** [comment]
Hey @AlexPerson  some good work here - some tips to help you improve

* The Gemfile can lock the project to a particular version of Ruby.  In general you should prefer the latest, currently 2.2.3

* Please do include all the gems you use in your Gemfile. This is an important courtesy to other developers and yourself in the future, so that all the project dependencies can be pulled in whenever the project is checked out on a new machine, e.g. `twilio-ruby` gem

* Please do update your README following the [contribution notes](https://github.com/makersacademy/takeaway-challenge/blob/master/CONTRIBUTING.md), i.e.

  * Make sure you have written your own README that briefly explains your approach to solving the challenge.
  * If your code isn't finished it's not ideal but acceptable as long as you explain in your README where you got to and how you would plan to finish the challenge.

* The above is a relatively straightforward thing to do that doesn't involve much programming - I'll often get it done while thinking about other problems in the back of my mind :-)

* Also it's a great idea to show the full story of how your app is used (from a user's perspective) in the README, e.g. an irb transcript

```sh
$ irb
Error : 'load /Users/tansaku/.irbrc' : The From phone number +441908410626 is not a valid, SMS-capable inbound phone number or short code for your account.
2.2.3 :001 > takeaway = Takeaway.new
 => #<Takeaway:0x007fac93bfadc8 @menu={"Cheeseburger"=>1, "Pizza"=>2, "Fries"=>3}, @actualcost=0> 
2.2.3 :002 > takeaway.takeorder({"Cheeseburger" => 1}, 1)
 => {"Cheeseburger"=>1} 
2.2.3 :003 > takeaway.actualcost
 => 1 
2.2.3 :004 > 
```

Note how I was unsure how to use your application from irb.  I tried to infer its operation from the tests and ran the above.

* You should be able to use just `require` in your specs - no need for `require_relative`

* note that tests like this:

```ruby
	it 'can place orders when passed a dish, quantity of meals, and total cost' do
		expect(subject).to respond_to(:takeorder).with(2).arguments
	end
```
can be collapsed to one liner like this

```ruby
    it { is_expected.to respond_to(:takeorder).with(2).arguments }
```

and also that these become somewhat redundant once you are actively testing the method like so:

```ruby
	it 'remembers orders' do
		subject.takeorder({"Cheeseburger" => 1}, 1)
		expect(subject.order).to eq ({"Cheeseburger" => 1})
	end
```
The respond_to tests are an initial step you go through using the tests to drive the creation of the objects public interface, and can safely be deleted once you have more sophisticated tests that check both the interface methods and their responses (and associated changes in state)

* Otherwise for the time beingI'd just backup the points made by @ojlamb and hound/rubocop

Overall this is a good base.  Don't be intimidated by the above list, you've done really well, but I want to give you all the help I can to take things to the next level :-)  There are a few other things as well, but have a go at fixing up some of the above, we'll do another round of feedback and then and we'll see about getting you a sticker :-)

:construction:

**Pull request closed without merging**